packages/wmr/src/lib/compile-single-module.js

```diff
@@ -27,55 +27,59 @@ let cache;
  * @param {string} options.out
  * @param {boolean} [options.hmr]
  * @param {boolean} [options.rewriteNodeImports]
+ * @param {import('rollup').ModuleFormat} [options.format]
  */
-export const compileSingleModule = withCache(async (input, { cwd, out, hmr = true, rewriteNodeImports = true }) => {
-	input = input.replace(/\.css\.js$/, '.css');
-	// console.log('compiling ' + input);
-	const bundle = await rollup.rollup({
-		input,
-		treeshake: false,
-		preserveModules: true,
-		// these should theroetically improve performance:
-		inlineDynamicImports: false,
-		preserveEntrySignatures: 'strict',
-		cache,
-		// perf: true,
-		// external: () => true,
-		plugins: [
-			{
-				name: 'wmr-single-file-resolver',
-				resolveId(id) {
-					if (id == input) return null;
-					// hmr client
-					if (id === 'wmr') id = '/_wmr.js';
-					// bare specifier = npm dep
-					else if (rewriteNodeImports && !/^\.?\.?(\/|$)/.test(id)) id = `/@npm/${id}`;
-					// relative imports (css)
-					else if (/\.css$/.test(id)) id += '.js';
-					return { id, external: true, moduleSideEffects: true };
-				}
-			},
-			sucrasePlugin({
-				typescript: true,
-				sourcemap: false,
-				production: false
-			}),
-			// localNpmPlugin(),
-			// wmrStylesPlugin({ cwd }),
-			hmr && wmrPlugin(),
-			htmPlugin()
-		].filter(Boolean)
-	});
-	cache = bundle.cache;
-	const result = await bundle.write({
-		...ROLLUP_FAST_OUTPUT,
-		dir: out,
-		assetFileNames: '[name].[ext]',
-		paths: str => str,
-		format: 'es'
-	});
-	return result.output[0].code;
-});
+export const compileSingleModule = withCache(
+	async (input, { cwd, out, hmr = true, rewriteNodeImports = true, format = 'es' }) => {
+		input = input.replace(/\.css\.js$/, '.css');
+		// console.log('compiling ' + input);
+		const bundle = await rollup.rollup({
+			input,
+			treeshake: false,
+			preserveModules: true,
+			// these should theroetically improve performance:
+			inlineDynamicImports: false,
+			preserveEntrySignatures: 'strict',
+			cache,
+			// perf: true,
+			// external: () => true,
+			plugins: [
+				{
+					name: 'wmr-single-file-resolver',
+					resolveId(id) {
+						if (id == input) return null;
+						// hmr client
+						if (id === 'wmr') id = '/_wmr.js';
+						// bare specifier = npm dep
+						else if (rewriteNodeImports && !/^\.?\.?(\/|$)/.test(id)) id = `/@npm/${id}`;
+						// relative imports (css)
+						else if (/\.css$/.test(id)) id += '.js';
+						return { id, external: true, moduleSideEffects: true };
+					}
+				},
+				sucrasePlugin({
+					typescript: true,
+					sourcemap: false,
+					production: false
+				}),
+				// localNpmPlugin(),
+				// wmrStylesPlugin({ cwd }),
+				hmr && wmrPlugin(),
+				htmPlugin()
+			].filter(Boolean)
+		});
+		cache = bundle.cache;
+		const result = await bundle.write({
+			...ROLLUP_FAST_OUTPUT,
+			dir: out,
+			assetFileNames: '[name].[ext]',
+			paths: str => str,
+			format,
+			exports: 'auto'
```

Comment:
Rollup will spit out the following message if this is not set:

```
Entry module "wmr.config.ts" is implicitly using "default" export mode, which means for CommonJS output that its default export is assigned to "module.exports". For many tools, such CommonJS output will not be interchangeable with the original ES module. If this is intended, explicitly set "output.exports" to either "auto" or "default", otherwise you might want to consider changing the signature of "wmr.config.ts" to use named exports only.
```

`auto` should be fine for us, explicitly setting it silences the warning.